yt/geometry/oct_container.pyx

```diff
@@ -170,7 +170,7 @@ cdef class OctreeContainer:
                 pos[2] = self.DLE[2] + dds[2]/2.0
                 for k in range(self.nn[2]):
                     if self.root_mesh[i][j][k] == NULL:
-                        raise RuntimeError
+                        raise KeyError(i,j,k)
```

Comment:
I guess an `IndexError` would be slightly more appropriate

Reply:
No, I think `KeyError` is fine.  The issue is that we have constructed what amounts to a mapping, and that mapping has nothing in it for a given key.

Reply:
ok, makes sense